pybamm/parameters/lithium_ion_parameters.py

```diff
@@ -224,6 +224,73 @@ def _set_dimensional_parameters(self):
             "Initial concentration in electrolyte [mol.m-3]"
         )
 
+        # mechanical prapmeters
+        if isinstance(self, pybamm.particle_cracking.BaseCracking):
+            self.flag_mechanics = pybamm.Parameter("Flag mechanical effects")
+            # flag: 1 - activate mechanical effects; 0 - disable
+            self.nu_p = pybamm.Parameter("Positive electrode Poisson's ratio")
+            self.E_p = pybamm.Parameter("Positive electrode Young's modulus [Pa]")
+            self.c_p_0 = pybamm.Parameter(
+                "Positive electrode reference concentration for free of deformation [m3.mol-1]"
+            )
+            self.Omega_p = pybamm.Parameter(
+                "Positive electrode partial molar volume [m3.mol-1]"
+            )
+            self.nu_n = pybamm.Parameter("Negative electrode Poisson's ratio")
+            self.E_n = pybamm.Parameter("Negative electrode Young's modulus [Pa]")
+            self.c_n_0 = pybamm.Parameter(
+                "Negative electrode reference concentration for free of deformation [m3.mol-1]"
+            )
+            self.Omega_n = pybamm.Parameter(
+                "Negative electrode partial molar volume [m3.mol-1]"
+            )
+            self.l_cr_p_0 = pybamm.Parameter(
+                "Positive electrode initial crack length [m]"
+            )
+            self.l_cr_n_0 = pybamm.Parameter(
+                "Negative electrode initial crack length [m]"
+            )
+            self.w_cr = pybamm.Parameter("Negative electrode initial crack width [m]")
+            self.rho_cr_n_dim = pybamm.Parameter(
+                "Negative electrode number of cracks per unit area of the particle [m-2]"
+            )
+            self.rho_cr_p_dim = pybamm.Parameter(
+                "Positive electrode number of cracks per unit area of the particle [m-2]"
+            )
+            self.b_cr = pybamm.Parameter("Negative electrode Paris' law constant b")
+            self.m_cr = pybamm.Parameter("Negative electrode Paris' law constant m")
+            self.k_cr = pybamm.Parameter("Negative electrode cracking rate")
+            self.Eac_cr_n = pybamm.Parameter(
+                "Negative electrode activation energy for cracking rate [kJ.mol-1]"
+            )  # noqa
+            self.Eac_cr_p = pybamm.Parameter(
+                "Positive electrode activation energy for cracking rate [kJ.mol-1]"
+            )  # noqa
+            self.alpha_T_cell_dim = pybamm.Parameter(
+                "Cell thermal expansion coefficien [m.K-1]"
+            )
+            self.R_const = pybamm.constants.R
```

Comment:
you can already get this as `self.R` (defined on line 55)

Reply:
corrected